sdk/sdktypes/code_location.go

```diff
@@ -9,9 +9,10 @@ import (
 )
 
 const (
-	symbolSeparator = ","
-	pathSeparator   = ":"
-	rowColSeparator = "."
+	symbolSeparator  = ","
+	pathSeparator    = ":"
+	altPathSeparator = " "
```

Comment:
Why do we need this?

Reply:
it makes the coloring in vscode work better